ui/decredmaterial/modal.go

```diff
@@ -90,14 +91,37 @@ func (m *Modal) Layout(gtx layout.Context, widgets []layout.Widget) layout.Dimen
 							if m.padding == unit.Dp(0) {
 								return layout.UniformInset(m.padding).Layout(gtx, title)
 							}
-							return layout.UniformInset(unit.Dp(10)).Layout(gtx, title)
+
+							inset := layout.Inset{
+								Top:    unit.Dp(10),
+								Bottom: unit.Dp(10),
+							}
+							return inset.Layout(gtx, title)
 						}
 						return D{}
 					}),
 					layout.Rigid(func(gtx C) D {
-						return m.scroll.Layout(gtx, len(widgetFuncs), func(gtx C, i int) D {
-							gtx.Constraints.Min.X = gtx.Constraints.Max.X
-							return layout.UniformInset(unit.Dp(10)).Layout(gtx, widgetFuncs[i])
+						mTB := unit.Dp(10)
+						mLR := unit.Dp(0)
+						if m.padding == unit.Dp(0) {
+							mLR = mTB
+						}
+						inset := layout.Inset{
+							Top:    mTB,
+							Bottom: mTB,
+							Left:   mLR,
+							Right:  mLR,
+						}
+						if m.scrollbar {
+							return m.scroll.Layout(gtx, len(widgetFuncs), func(gtx C, i int) D {
+								gtx.Constraints.Min.X = gtx.Constraints.Max.X
+								return inset.Layout(gtx, widgetFuncs[i])
+							})
+						}
+						list := &layout.List{Axis: layout.Vertical}
+						gtx.Constraints.Min.X = gtx.Constraints.Max.X
+						return list.Layout(gtx, len(widgetFuncs), func(gtx C, i int) D {
+							return inset.Layout(gtx, widgetFuncs[i])
```

Comment:
I'm weary of extensive changes to existing widget layouts. I'll only ask if you've tested extensively to be sure that these changes do not result in unwanted UI effects in some places.

Reply:
Yes i have.. the changes makes adjustments to the padding and displays the scroll bar in modals that needs, which in this case is the account selector modal.